packages/back-end/src/services/experimentNotifications.ts

```diff
@@ -418,8 +418,8 @@ export const notifyExperimentChange = async ({
   const experiment = await getExperimentById(context, snapshot.experiment);
   if (!experiment) throw new Error("Error while fetching experiment!");
 
-  // do not fire significance or error events for dimension analyses
-  if (snapshot.dimension) return;
+  // do not fire significance or error events for ad-hoc analyses
+  if (snapshot.type === "ad-hoc") return;
```

Comment:
Also return if `snapshot.type` is not set (just in case this code runs for ad ad-hoc snapshot that is in Mongo from before the PR merges)

Reply:
done ty